src/cpu/x64/jit_avx512_core_bf16cvt.hpp

```diff
@@ -258,7 +259,7 @@ struct jit_avx512_core_add_cvt_ps_to_bf16_t : public jit_generator {
 private:
     int simd_w_;
 
-    bf16_emulation_t *bf16_emu_;
+    std::unique_ptr<bf16_emulation_t> bf16_emu_;
```

Comment:
Since there's no actual usage of the pointer here, we will want to manually check the cpp file or elsewhere in the codebase to make sure the usage is okay.

Of course, it compiles, so that gives me hope.